app/assets/javascripts/views/editor/editor_view.ts

```diff
@@ -381,38 +383,39 @@ class EditorViewCtrl extends PureViewCtrl implements EditorViewScope {
     this.removeComponentsObserver = this.application.streamItems(
       ContentType.Component,
       async (items) => {
-        const components = items as SNComponent[];
-        if (!this.note) {
-          return;
-        }
-        /** Reload componentStack in case new ones were added or removed */
-        await this.reloadComponentStack();
+        if (!this.note) return;
+        this.setState({
+          allStackComponents: sortAlphabetically(
```

Comment:
Should this just be called `stackComponents` instead of `all`? Because if previously it was called `activeStackComponents` then the new naming would not be `all`, but a subset of all that are active.